### PR TITLE
Move indemnification warnings to templates

### LINKED ISSUE #316: Indemnification not set warning appears randomly with no indication of which organization
_Please add `bug`, the name of any relevant modules (e.g. `rocky`), and any other relevant labels to your issue._

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**
image

image

**OpenKAT version**
Note the release tag (and if possible: the installation method) here.
If it concerns an in-development version, note the branch(es) and commit hash(es) here as well.

**Desktop (please complete the following information if relevant):**
 - Browser: [e.g. chrome, safari]
 - Version: [e.g. 22]

**Smartphone (please complete the following information if relevant):**
 - Browser: [e.g. stock browser, safari]
 - Version: [e.g. 22]

**Additional context**

### PR DESCRIPTION
**Pull request by praseodym**
## Changes

This will prevent the warning from appearing randomly.

## Issue ticket number and link

Fixes #316

## Proof

Crisis room:
Screenshot 2023-04-11 at 14-50-16 OpenKAT - organization_crisis_room

Organization settings:
Screenshot 2023-04-11 at 14-50-24 OpenKAT - organization_settings

## Extra instructions for others
_This section may be skipped or omitted. Uncomment and answer the below questions if relevant._

## Checklist for author(s):
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR comes from a `feature` or `hotfix` branch, in line with our git branching strategy;
- [x] This PR is "bite-sized" and only focuses on a single issue, problem, or feature;
- [x] I am not reinventing the wheel: there is no high-quality library that already has this feature;
- [x] I have changed the example `.env` files if I added, removed, or changed any config options, and I have informed others that they need to modify their `.env` files if required;
- [x] I have performed a self-review of my own code;
- [x] I have commented my code, particularly in hard-to-understand areas;
- [ ] I have made corresponding changes to the documentation, if necessary;
- [ ] I have written unit, integration, and end-to-end tests for the change that I made;

If a non-trivial PR:
- [ ] This PR is part of a milestone and has appropriate labels;
- [ ] This PR is properly linked to the project board (either directly or via an issue);
- [ ] I have added screenshots or some other proof that my code does what it is supposed to do;

```
## Checklist for functional reviewer(s):
- [ ] If a non-trivial PR: This PR is properly linked to an issue on the project board;
- [ ] I have checked out this branch, and successfully ran `make kat`;
- [ ] I have ran `make test-rf` and all end-to-end Robot Framework tests pass;
- [ ] I confirmed that the PR's advertised `feature` or `hotfix` works as intended;
- [ ] I confirmed that there are no unintended functional regressions in this branch;

### What works:
* _bullet point + screenshot (if useful) per tested functionality_

### What doesn't work:
* _bullet point + screenshot (if useful) per tested functionality_

### Bug or feature?:
* _bullet point + screenshot (if useful) if it is unclear whether something is a bug or an intended feature._
```

```
## Checklist for code reviewer(s):
- [ ] The code passes the CI tests and linters;
- [ ] The code does not bypass authentication or security mechanisms;
- [ ] The code does not introduce any dependency on a library that has not been properly vetted;
- [ ] The code does not violate Model-View-Template and our other architectural principles;
- [ ] The code contains docstrings, comments, and documentation where needed;
- [ ] The code prioritizes readability over performance where appropriate;
- [ ] The code conforms to our agreed coding standards.
```

### CODE REVIEW

File: rocky/rocky/templates/organizations/organization_crisis_room.html [hunk 1]
```diff
@@ -8,76 +8,91 @@
...
+                                        </tr>
+                                        <tr>
+                                            <td>{% translate "OOI type:" %}</td>
+                                            <td>
+                                                <a href="{% url "ooi_list" organization_code=organization.code %}?ooi_type={{ ooi.finding_type.object_type }}"
+                                                   title="{% blocktranslate trimmed with ooi_type=ooi.finding_type.object_type %} Show {{ ooi_type }} objects {% endblocktranslate %}">{{ ooi.finding_type.object_type }}</a>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>{% translate "Source OOI:" %}</td>
```
Comment (ammar92) [inline rocky/rocky/templates/organizations/organization_crisis_room.html:80]:
```suggestion
                                            <td>{% translate "Source OOI" %}:</td>
```

File: rocky/rocky/templates/organizations/organization_crisis_room.html [hunk 2]
```diff
@@ -8,76 +8,91 @@
...
+                                            data-close-label="{% translate "Close details" %}">
+                                        {% translate "Open details" %}
+                                    </button>
+                                </td>
+                            </tr>
+                            <tr class="expando-row">
+                                <td colspan="6">
+                                    <table>
+                                        <tr>
+                                            <td>{% translate "Finding type:" %}</td>
```
Comment (ammar92) [inline rocky/rocky/templates/organizations/organization_crisis_room.html:66]:
```suggestion
                                            <td>{% translate "Finding type" %}:</td>
```

File: rocky/rocky/templates/organizations/organization_crisis_room.html [hunk 3]
```diff
@@ -8,76 +8,91 @@
...
+                                            <td>
+                                                <a href="{% ooi_url "ooi_detail" ooi.finding_type organization.code query=mandatory_fields %}"
+                                                   title="{% blocktranslate trimmed with finding_type=ooi.finding_type.human_readable %} Show details for {{ finding_type }} {% endblocktranslate %}">{{ ooi.finding_type.human_readable }}</a>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>{% translate "OOI type:" %}</td>
```
Comment (ammar92) [inline rocky/rocky/templates/organizations/organization_crisis_room.html:73]:
```suggestion
                                            <td>{% translate "OOI type" %}:</td>
```

**Review by ammar92 — APPROVED** [review]
Looks good 👍 Small remark for consistency in the translations

**Comment by praseodym** [comment]
> Looks good 👍 Small remark for consistency in the translations

I haven't changed anything to those lines except for whitespace, do we still want to make these changes in this PR?

**Comment by praseodym** [comment]
> > Looks good 👍 Small remark for consistency in the translations
> 
> I haven't changed anything in those lines except for whitespace, do we still want to make these changes in this PR?

Done 😃

**Review by ammar92 — APPROVED** [review]

**Pull request merged**